### PR TITLE
Add new network module qnos_command to configure Quanta switches

### PR DESCRIPTION
**Pull request by Quanta-Switch**
##### SUMMARY
Add network module qnos_command, which is based on ios_command

This PR is based on add new platform for managing Quanta Switches #59569

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
lib/ansible/modules/network/qnos/qnos_command.py

test/units/modules/network/qnos/test_qnos_command.py
test/units/modules/network/qnos/fixtures/show_version
test/units/modules/network/qnos/fixtures/configure

##### ADDITIONAL INFORMATION

```paste below
ansible_network_os=qnos
ansible_connection=network_cli
```

### CODE REVIEW

**Comment by pabelanger** [comment]
This will need to be rebased, as there is a merge conflict.

File: test/sanity/code-smell/action-plugin-docs.py [hunk 1]
```diff
@@ -1,39 +0,0 @@
-#!/usr/bin/env python
```
Comment (mattclay) [inline test/sanity/code-smell/action-plugin-docs.py]: This file shouldn't be removed.

Comment (Quanta-Switch) [reply test/sanity/code-smell/action-plugin-docs.py]: Already add back, i.e. this file is not changed.

File: test/sanity/code-smell/action-plugin-docs.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```
Comment (mattclay) [inline test/sanity/code-smell/action-plugin-docs.py]: This file should not be added.

**Review by mattclay — CHANGES_REQUESTED** [review]

**Comment by samccann** [comment]
Hi @quantasw.  Unfortunately we are no longer accepting new content submissions into the Ansible project, but instead you should create an Ansible Content Collection and publish it on Ansible Galaxy. 

Developing a Collection: https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html#developing-collections

Creating a Galaxy namespace: https://galaxy.ansible.com/docs/contributing/namespaces.html

Hope this helps!

**Comment by quantasw** [comment]
Hi @samccann  The purpose of closing this PR is that our PR is not fitting your SPEC or PR request method is changed to use "Ansible-galaxy"? Our target is putting ansible setting of our product to ansible official website so that customers can use it all the time.

**Comment by samccann** [comment]
Hi @quantasw - The PR is closed because yes, new modules need to be in a collection format. You can upload that collection to Ansible Galaxy. In the near term future, most modules will be moved out of this ansible/ansible repo and be placed as part of a collection in Galaxy.  The Ansible core team meets at 10 am ET today on IRC #ansible-meeting if you have further questions etc. Or if you can't make that timeslot, you can ask questions any time on #ansible-devel .

**Pull request closed without merging**